.travis.yml

```diff
@@ -26,10 +26,10 @@ install:
   - conda install -c conda-forge aipy
   - pip install coveralls
   - pip install git+https://github.com/HERA-Team/pyuvdata.git
-  - pip install git+https://github.com/HERA-Team/hera_cal.git
+  - python setup.py install  # Needs to be installed before hera_cal
```

Comment:
I agree this is a pain. It's not due to datacontainer though, it's because hera_cal uses ant_metrics. I think it just uses it to read in the metrics to flag bad ants, so we could potentially move that import to within a function, then we can put these back in order.